### PR TITLE
PR for a lot of changes

### PR DESCRIPTION
**Pull request by Camouflage100**
* WebStorm auto code style/reformat
* Cleaned up frontend
* Fixed sidebar navbar spacing issue with the top of the navigation going into the user's background image.
* Placeholder image gets the player's in-game skin's head.
-- Added credits to Crafatar on the credits page.
* Added safe check for plugin viewing on "fake plugins"
-- https://bstats.org/plugin/bukkit/_bukkit_ will auto redirect to https://bstats.org/global/bukkit

### CODE REVIEW

**Comment by ljacqu** [comment]
-1.

Why do you add .idea/ files? Should go into gitignore. 
Also you added reformatted the .min.js file.

File: views/static/index.ejs [hunk 1]
```diff
@@ -89,7 +90,8 @@
                         <h2 class="center red-text"><i class="material-icons">code</i></h2>
                         <h5 class="center">Open source</h5>
 
-                        <p class="light">bStats is Open Source! Feel free to check out our <a href="https://github.com/BtoBastian/bStats">GitHub page</a>.</p>
+                        <p class="light">bStats is Open Source! Feel free to check out our <a
+                                    href="https://github.com/BtoBastian/bStats">GitHub page</a>.</p>
```
Comment (ljacqu) [inline views/static/index.ejs]: Why did you remove the space between `!` and `You`?

**Comment by MiniDigger** [comment]
such huge PRs are never a good idea. your formatting changes make the diff unreadable and hide the actual changes. 
stuff like the .idea dir or iml files are system specific and go into your global git ignore. 
I would suggest splitting up this pr into multiple or at least multiple commits.

File: views/static/index.ejs [hunk 2]
```diff
@@ -59,8 +60,8 @@
                         <h2 class="center red-text"><i class="material-icons">layers</i></h2>
                         <h5 class="center">Hosted by LayerWall</h5>
 
-                        <p class="light">We are glad to be sponsored by <a href="http://www.layerwall.com/">LayerWall</a>!
-                            You should definitely check out their website!</p>
+                        <p class="light">&nbsp;We are glad to be sponsored by <a href="http://www.layerwall.com/">LayerWall</a>!You
+                            should definitely check out their website!</p>
```
Comment (ljacqu) [inline views/static/index.ejs]: Why did you remove the space between `!` and `You`?

**Comment by Camouflage100** [comment]
WebStorm did that automatically, my bad.  And yeah didn't think to do that.

**Comment by Camouflage100** [comment]
Auto format.

**Comment by Camouflage100** [comment]
I'll keep that in mind, sorry.

**Comment by MiniDigger** [comment]
I am pretty sure bastian would appreciate your contributions, but if you mix formatting and actual changes, its just impossible to see whats actually being changed.
if you could open a new PR with multiple commits that would be great!
